frontend/src/components/auth/Login.tsx

```diff
@@ -82,35 +103,58 @@ const Login = (): React.ReactElement => {
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
-          >
-            Log In
-          </button>
-        </div>
-        <GoogleLogin
+    <Flex h="100vh">
+      <Box w="50%">
+        <Image src={KidsPicture} />
+      </Box>
+      <Flex w="50%" justifyContent="center">
+        <Flex
+          direction="column"
+          alignItems="center"
+          justifyContent="center"
+          maxWidth="450px"
+        >
+          <Image src={JumpMathLogo} alt="Jump Math Logo" w="296px" mb="24px" />
+          <Text textStyle="header4" mb="24px">
+            Admin Login
+          </Text>
+          <Text textStyle="subtitle2" mb={loginError ? "12px" : "48px"}>
+            Enter your login credentials below to continue
+          </Text>
+          {loginError && (
+            <Flex alignItems="center" pb="12px">
+              <AlertCircle />{" "}
+              <Text color="red.200" pl="5px">
+                Please ensure fields are filled
+              </Text>
+            </Flex>
+          )}
+          <FormControl textStyle="subtitle2" isRequired>
+            <Flex>
+              <Text textStyle="subtitle2">Email Address</Text>
+              <FormLabel />
+            </Flex>
+            <Input
+              type="email"
+              value={email}
+              onChange={(e) => setEmail(e.target.value)}
+              mb="24px"
+              placeholder="Enter Jump Math email address"
+              variant="filled"
+            />
+            <Flex>
+              <Text textStyle="subtitle2">Password</Text>
+              <FormLabel />
+            </Flex>
+            <Input
+              type="password"
+              value={password}
+              onChange={(e) => setPassword(e.target.value)}
+              mb="32px"
+              placeholder="Enter Password"
+              variant="filled"
+            />
+            {/* <GoogleLogin
```

Comment:
please also remove all the Google login related code in this file! we won't be using it in our site